File: cpp/cpp_1000-/1450.cc
// C++
// leetcode 1450
// https://leetcode.cn/problems/number-of-students-doing-homework-at-a-given-time/

class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
    	int n = startTime.size(), res = 0;
    	for (int i = 0; i < n; i++) {
    		if (startTime[i] <= queryTime && endTime[i] >= queryTime)
    			res++;
    	}
    	return res;
    }
};

File: Go/Go_1000-/1450.go
// Go
// leetcode 1450
// https://leetcode.cn/problems/number-of-students-doing-homework-at-a-given-time/

func busyStudent(startTime []int, endTime []int, queryTime int) (res int) {
	n := len(startTime)
	for i := 0; i < n; i++ {
		if startTime[i] <= queryTime && queryTime <= endTime[i] {
			res++
		}
	}
	return res
}